aleph/search/facet.py

```diff
@@ -93,12 +93,17 @@ def update(self, result, key):
         result["label"] = key if event is None else event.title
 
 
-class DateFacet(Facet):
-    def update(self, result, key):
-        result["label"] = datetime.strptime(key, "%Y-%m-%dT%H:%M:%S").date()
+class EntityFacet(Facet):
+    def expand(self, keys):
+        for key in keys:
+            resolver.queue(self.parser, Entity, key)
+        resolver.resolve(self.parser)
 
-    def get_key(self, bucket):
```

Comment:
Are there any issues with removing this function?

Reply:
No, it's part of the DateFacet class we're no longer using.